src/Symfony/Component/HttpFoundation/Tests/AcceptHeaderTest.php

```diff
@@ -109,7 +111,12 @@ public static function provideSortingData()
     public function testDefaultValue($acceptHeader, $value, $expectedQuality)
     {
         $header = AcceptHeader::fromString($acceptHeader);
-        $this->assertSame($expectedQuality, $header->get($value)->getQuality());
+        $item = $header->get($value);
+        if (null === $expectedQuality) {
+            $this->assertNull($item);
+        } else {
+            $this->assertSame($expectedQuality, $item->getQuality());
+        }
```

Comment:
```suggestion
        $this->assertSame($expectedQuality, $item->getQuality());
```

Reply:
Thanks, @nicolas-grekas ! I've applied the suggestions